app.js

```diff
@@ -125,19 +126,19 @@ function generateRecipe(data) {
                 ${randomRecipe.ingredients.map(elmt => `
                 <li>${elmt.name} ${elmt.quantity}</li>
                 `).join('')}
-            </ul>
+            </ul
         </div>
     </div>
     `;
-    main.insertAdjacentHTML('beforeend', html3);
+    main.insertAdjacentHTML('beforeend', html3); // Comment on naming
     header.insertAdjacentHTML('beforeend', '<a href="index.html">New Search</a>');
 }
 
 //Event Listeners ---------------------------------------------------------------------------------------------------------------------
 btnSearch.addEventListener('click', () => {
-    let regEx = /[a-zA-Z]/;
+    let regEx = /[a-zA-Z]/; // Comment on regex and errors
```

Comment:
Why are you limiting show names to only letters?  What about using the word [character class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes) to match letters and numbers?